apps/builder/app/builder/shared/sync/sync-server.ts

```diff
@@ -305,6 +311,22 @@ const useSyncProject = ({
   }, [projectId, authPermit]);
 };
 
+export class ServerSyncStorage implements SyncStorage {
+  name = "server";
+  sendTransaction(transaction: Transaction<Change[]>) {
+    if (transaction.object === "server") {
+      pendingTransactions.push(transaction);
```

Comment:
I have feeling that we can just do
`commandQueue.enqueue({ type: "transactions", transactions, projectId });`
here

Not getting why we have setInterval

Reply:
See in poll logic we already have sleep insterval. There is no need in intermediate pending with additional interval

Reply:
Indeed